integrationExamples/gpt/permutiveRtdProvider_example.html

```diff
@@ -135,15 +141,27 @@
             pbjs.que.push(function() {
               pbjs.setConfig({
                 debug: true,
+                pageUrl: 'http://www.test.com/test.html',
                 realTimeData: {
                   auctionDelay: 80, // maximum time for RTD modules to respond
                   dataProviders: [
                     {
                       name: 'permutive',
                       waitForIt: true,
                       params: {
-                        acBidders: ['appnexus', 'rubicon', 'ozone', 'trustx'],
+                        acBidders: ['appnexus', 'rubicon', 'ozone', 'trustx', 'ix'],
                         maxSegs: 500,
+                        transformations: [
+                          {
+                            id: 'iabAudienceTaxonomy',
```

Comment:
I don't think this is precise enough; there are multiple IAB taxonomies

Reply:
only 1 Audience taxonomy I think? 
https://iabtechlab.com/standards/

Reply:
Sorry for not being clear -- I was thinking perhaps a user of the module should be able to configure the version used?

Reply:
Yep, fair enough